test/Conversion/DmaToIpu/dma_to_ipu_issue_token.mlir

```diff
@@ -0,0 +1,40 @@
+//===- dma_to_ipu_issue_token.mlir -----------------------------*- MLIR -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2024 Advanced Micro Devices, Inc.
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: aie-opt -aie-dma-to-ipu %s | FileCheck %s
+
+// TODO - more
+// CHECK-LABEL: test1
+// CHECK: aiex.ipu.writebd_shimtile
+// CHECK-SAME: ddr_id = 0 : i32
+// CHECK-SAME: valid_bd = 1 : i32
+// CHECK: aiex.ipu.write32
+// CHECK-SAME: value = 2147483649
+// CHECK: aiex.ipu.writebd_shimtile
+// CHECK-SAME: ddr_id = 1 : i32
+// CHECK: aiex.ipu.write32
+// CHECK-SAME: value = 0
+module  {
+aie.device(ipu) {
+  memref.global "public" @toMem : memref<16xi32>
+  memref.global "public" @fromMem : memref<16xi32>
+  func.func @test1(%arg0: memref<16xi32>, %arg1: memref<16xi32>) {
+    %c16_i64 = arith.constant 16 : i64
+    %c64_i64 = arith.constant 64 : i64
+    %c0_i64 = arith.constant 0 : i64
+    %c1_i64 = arith.constant 1 : i64
```

Comment:
also just btw you don't need these as `arith.constant` - [you can just "inline" the literals](https://github.com/Xilinx/mlir-aie/blob/b334a0708021f0f8bbf06d9ea8916b7bebbb8d69/test/Targets/AIETargetHSA/input_with_addresses.mlir#L49)

Reply:
Thanks for the pointer! I inlined the literals.